### PR TITLE
Quickly get a logger with IntelliJ live templates

### PR DESCRIPTION
**Pull request by malteo**

### CODE REVIEW

File: website/blog/2018-04-18-intellij-logger-live-template.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+- Open **Edit variables** and write in the field *Expression* at the `CLASS` row
+
+      className()
+
+  and press OK
+
+- Set the applicable context by clicking on `Java` in the **No applicable contexts yet. Define** popup
+
+Exit the settings by pressing OK.
```
Comment (pimuzzo) [inline website/blog/2018-04-18-intellij-logger-live-template.md:53]: is it better to press apply and then ok?

File: website/blog/2018-04-18-intellij-logger-live-template.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+---
+title:     "Quickly get a logger with IntelliJ live templates"
```
Comment (gionn) [inline website/blog/2018-04-18-intellij-logger-live-template.md:2]: s/get/define

File: website/blog/2018-04-18-intellij-logger-live-template.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+author:    Matteo Giordano
+authorURL: http://twitter.com/malteo
+---
+
+IntelliJ IDEA is every Java developer's favourite IDE here at Cloudesire, and amongst its powerful features there are [Live templates](https://www.jetbrains.com/help/idea/creating-and-editing-live-templates.html). Let me show you how they work with a simple example.
+
+Getting a [SLF4J](https://www.slf4j.org/) instance for the current class is a matter of typing
```
Comment (gionn) [inline website/blog/2018-04-18-intellij-logger-live-template.md:9]: :

File: website/blog/2018-04-18-intellij-logger-live-template.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+```java
+private static final Logger
+```
+
+waiting for the import popup to appear, looking for `org.slf4j.Logger` and selecting it, continue typing
```
Comment (gionn) [inline website/blog/2018-04-18-intellij-logger-live-template.md:15]: :

File: website/blog/2018-04-18-intellij-logger-live-template.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
+---
+title:     "Quickly get a logger with IntelliJ live templates"
+author:    Matteo Giordano
+authorURL: http://twitter.com/malteo
+---
+
+IntelliJ IDEA is every Java developer's favourite IDE here at Cloudesire, and amongst its powerful features there are [Live templates](https://www.jetbrains.com/help/idea/creating-and-editing-live-templates.html). Let me show you how they work with a simple example.
+
```
Comment (gionn) [inline website/blog/2018-04-18-intellij-logger-live-template.md:8]: `<!--truncate-->` here

File: website/blog/2018-04-18-intellij-logger-live-template.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+Getting a [SLF4J](https://www.slf4j.org/) instance for the current class is a matter of typing
+
```
Comment (gionn) [inline website/blog/2018-04-18-intellij-logger-live-template.md:10]: `<!--truncate-->` or here

**Review by gionn — APPROVED** [review]

**Pull request merged**